docs/installation/config-reference.rst

```diff
@@ -512,3 +512,265 @@ Additional settings that are optional and maybe passed to Replicator Executable
+    
+
+``KSQL_CONSUMER_INTERCEPTOR_CLASSES``
+    
+
+  .. sourcecode:: bash
+
+    docker run -d \
+      -p 127.0.0.1:8088:8088 \
+      -e KSQL_BOOTSTRAP_SERVERS=localhost:9092 \
+      -e KSQL_LISTENERS=http://0.0.0.0:8088/ \
+      -e KSQL_KSQL_SERVICE_ID=confluent_test_2_ \
+      -e KSQL_PRODUCER_INTERCEPTOR_CLASSES=io.confluent.monitoring.clients.interceptor.MonitoringProducerInterceptor \
+      -e KSQL_CONSUMER_INTERCEPTOR_CLASSES=io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor \
+      confluentinc/cp-ksql-server:5.0.0
+
+
+In interactive mode, the cli running outside Docker can connect to the server running in Docker:
+
+  .. sourcecode:: bash
+
+    ./bin/ksql
+    ... Edited out the ascii art...
+    CLI v5.0.0, Server v5.0.0-SNAPSHOT located at http://localhost:8088
+
+    Having trouble? Type 'help' (case-insensitive) for a rundown of how things work!
+
+    ksql>
+
+Connect to a Secure Kafka Cluster, Like |ccloud|
+""""""""""""""""""""""""""""""""""""""""""""""""
+
+``KSQL_BOOTSTRAP_SERVERS``
+    List of hosts that are running in a Kafka cluster. 
+
+``KSQL_KSQL_SERVICE_ID``
+    The ID...
+
+``KSQL_LISTENERS``
+    
+``KSQL_KSQL_SINK_REPLICAS``
+    
+
+``KSQL_KSQL_STREAMS_REPLICATION_FACTOR``
+    
+
+``KSQL_SECURITY_PROTOCOL``
+    
+
+``KSQL_SASL_JAAS_CONFIG``
+    
+
+  .. sourcecode:: bash
+
+    docker run -d \
+      -p 127.0.0.1:8088:8088 \
+      -e KSQL_BOOTSTRAP_SERVERS=REMOVED_SERVER1:9092,REMOVED_SERVER2:9093,REMOVED_SERVER3:9094 \
+      -e KSQL_LISTENERS=http://0.0.0.0:8088/ \
+      -e KSQL_KSQL_SERVICE_ID=default_ \
+      -e KSQL_KSQL_SINK_REPLICAS=3 \
+      -e KSQL_KSQL_STREAMS_REPLICATION_FACTOR=3 \
+      -e KSQL_SECURITY_PROTOCOL=SASL_SSL \
+      -e KSQL_SASL_MECHANISM=PLAIN \
+      -e KSQL_SASL_JAAS_CONFIG="org.apache.kafka.common.security.plain.PlainLoginModule required username=\"...\" password=\"...\";" \
+      confluentinc/cp-ksql-server:5.0.0
+
+
+Configuration via Java system properties
+""""""""""""""""""""""""""""""""""""""""
+
+``KSQL_BOOTSTRAP_SERVERS``
+    List of hosts that are running in a Kafka cluster. 
+
+``KSQL_OPTS``
+    
+
+
+  .. sourcecode:: bash
+
+    docker run -d \
+      -v /path/on/host:/path/in/container/ \
+      -e KSQL_BOOTSTRAP_SERVERS=localhost:9092 \
+      -e KSQL_OPTS="-Dksql.service.id=confluent_test_3_  -Dksql.queries.file=/path/in/container/queries.sql" \
+      confluentinc/cp-ksql-server:5.0.0
+
+
+View logs
+"""""""""
+
+  .. sourcecode:: bash
+
+    docker logs -f <container id>
+    [2018-05-24 23:43:05,591] INFO stream-thread [_confluent-ksql-default_transient_1507119262168861890_1527205385485-71c8a94c-abe9-45ba-91f5-69a762ec5c1d-StreamThread-17] Starting (org.apache.kafka.streams.processor.internals.StreamThread:713)
+    ...
+
+
+--------
+KSQL CLI
+--------
+
+Connect to a Dockerized KSQL Server
+"""""""""""""""""""""""""""""""""""
+
+This works because of the Docker network created by the KSQL Server.
```

Comment:
suggest: `The Docker network created by the KSQL Server enables you to connect to a dockerized KSQL server.`